### DISCUSSION TITLE
PHP COMPILED FROM SOURCE: PHP WARNINGS

### DISCUSSION THREAD

**Discussion by farrukhahmed4220** [oerdnj/deb.sury.org#2170] [General]
Could you help me? I'm getting this warning `PHP Warning:  PHP Startup: Invalid library (maybe not a PHP library)`  when I create an extension ini file in `/etc/php/7.4/mods-available` and symlink it inside `/etc/php/7.4/cli/conf.d` OR `/etc/php/7.4/fpm/conf.d`

But when I remove the ini files, the warning disappears.

Any Idea how to resolve this?

**Comment by oerdnj** [comment]
> PHP COMPILED FROM SOURCE

I am sorry, but why are you asking here?
